Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/properties/PropertyAccessor.java

```diff
@@ -116,7 +116,7 @@ public synchronized String getProperty(String propertyFile, String propertyName,
      * @throws PropertyAccessException the property access exception
      */
     @Override
-    public void setProperty(String propertyFileName, String key, String value) throws PropertyAccessException {
+    public synchronized void setProperty(String propertyFileName, String key, String value) throws PropertyAccessException {
```

Comment:
Question: do we need to use synchronized?

Reply:
Sync'ing the setProperty method will avoid having dirty reads while we are writing the file to disk. I am not sure why the Setter was the only method in this class that was not synchronized - if there was anything here that wouldn't need to be synchronized, it would be the getter.